javascript-data-model/order-history.js

```diff
@@ -1,69 +1,53 @@
 var orderHistory = [
   {
-    item: 'Javascript for impatient programmers',
-    author: 'Rauschmayer, Dr. Axel',
-    itemCost: 31.55,
+    item: ['Javascript for impatient programmers'],
+    author: ['Rauschmayer, Dr. Axel'],
+    itemCost: [31.55],
     orderTotal: 34,
     orderPlaced: 'August 4, 2020',
     delivered: 'August 8, 2020',
     returnWindow: 'September 7, 2020',
-    shipto: 'JS Masher',
+    shipTo: 'JS Masher',
     orderNumber: '114-394',
-    image: 'url javascript'
+    image: ['url javascript']
   },
   {
-    item: 'The Timeless Way of Building',
-    author: 'Alexander, Christopher',
-    itemCost: 41.33,
+    item: ['The Timeless Way of Building'],
+    author: ['Alexander, Christopher'],
+    itemCost: [41.33],
     orderTotal: 44.53,
     orderPlaced: 'July 19, 2020',
     delivered: 'July 20, 2020',
     returnwindow: 'August 19, 2020',
-    shipto: 'JS Masher',
+    shipTo: 'JS Masher',
     orderNumber: '113-998',
-    image: 'url the timeless way of building'
+    image: ['url the timeless way of building']
   },
   {
-    item: 'Gamecube Controller Adapter',
-    author: ' ',
-    itemCost: 17.22,
+    item: ['Gamecube Controller Adapter'],
+    author: [' '],
+    itemCost: [17.22],
     orderTotal: 15.98,
     orderPlaced: 'July 4, 2020',
     delivered: 'July 7, 2020',
     returnWindow: 'August 5, 2020',
     shipTo: 'JS Masher',
     orderNumber: '114-287',
-    image: 'url gamecube controller adapter'
+    image: ['url gamecube controller adapter']
   },
   {
-    itemBatch: [
-      {
-        item: 'Gamecube Controller',
-        author: ' ',
-        itemCost: 94.95,
-        orderTotal: 138.93,
-        orderPlaced: 'July 3, 2020',
-        delivered: 'July 5, 2020',
-        returnWindow: 'August 4, 2020',
-        shipTo: 'JS Masher',
-        orderNumber: '113-288',
-        image: 'url gamecube controller'
-      },
-      {
-        item: 'The Art of Sql',
-        author: 'Faroult, Stephane',
-        itemCost: 33.99,
-        orderTotal: 138.93,
-        orderPlaced: 'July 3, 2020',
-        delivered: 'July 5, 2020',
-        returnWindow: 'August 4, 2020',
-        shipTo: 'JS Masher',
-        orderNumber: '113-288',
-        image: 'url The Art of Sql'
-      }
-    ]
+    item: ['Gamecube Controller', 'The Art of Sql'],
+    author: [' ', 'Faroult, Stephane'],
```

Comment:
Rather than having individual arrays for each property, I suggest having one array with an object for each item within it.